caramel/scripts/generate_ca.py

```diff
@@ -194,9 +198,14 @@ def build_ca(keyname, certname):
 
 def main():
     args = cmdline()
-    env = bootstrap(args.inifile)
+    config_uri = args.inifile
```

Comment:
That's a path, not an URI

Reply:
initializedb has it as config_uri, and to keep consistency I used what was already established convention.

Reply:
Just because other things do it wrong doesn't mean we should continue doing it wrong.   initializedb had it only because it used to be an URI, and was turned to a path.  It's ok to fix in both places.